packages/components/src/components/scrollable/scrollable.stories.ts

```diff
@@ -43,148 +47,147 @@ export default {
-  }
+/**
+ * Use the `scrollbars` attribute to display browser scrollbars.
+ */
+
+export const Scrollbar = {
+  name: 'Browser Scrollbar',
+  render: () => html`
+    <sd-scrollable orientation="vertical" buttons scrollbars>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
+        <p>It contains multiple paragraphs and lines.</p>
+        <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+        <p>The scrollable component will display shadows and buttons based on the props.</p>
+        <p>Customize the content and attributes as needed.</p>
+      </div>
+    </sd-scrollable>
+  `
 };
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">
-      ${generateTemplate({ args, constants: [{ type: 'attribute', name: 'buttons', value: true }] })}
-    </div>`;
-  },
+/**
+ * Use the `buttons` attribute to display buttons indicating scrollable content.
+ */
 
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-scrollable');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
+export const Buttons = {
+  name: 'Buttons',
+  render: () => html`
+    <sd-scrollable orientation="vertical" buttons>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
+        <p>It contains multiple paragraphs and lines.</p>
+        <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+        <p>The scrollable component will display shadows and buttons based on the props.</p>
+        <p>Customize the content and attributes as needed.</p>
+      </div>
+    </sd-scrollable>
+  `
+};
 
-    el?.shadowRoot?.querySelector<HTMLElement>('button')!.focus();
-  }
+/**
+ * Use the `shadows` attribute to display visual cues indicating scrollable content.
+ */
+
+export const Shadows = {
+  name: 'Shadows',
+  render: () => html`
+    <sd-scrollable orientation="vertical" shadows>
+      <div class="items-start h-max p-4 text-sm justify-start">
+        <p>Scroll vertically</p>
+        <br />
+        <p>This is a long scrollable content.</p>
+        <p>It contains multiple paragraphs and lines.</p>
+        <p>The content is intentionally long to trigger scrolling. You can scroll horizontally and vertically.</p>
+        <p>The scrollable component will display shadows and buttons based on the props.</p>
+        <p>Customize the content and attributes as needed.</p>
+      </div>
+    </sd-scrollable>
+  `
+};
+
+/**
+ * Use the `inset` attribute to adjust the inset padding.
+ */
+
+export const Inset = {
+  name: 'Inset',
+  render: () => html`
+    <sd-scrollable orientation="vertical" inset>
```

Comment:
Hard to see that inset is true ... pls add shadow bottom like in Figma .. then it is visible that the padding is larger.